apps/api/package.json

```diff
@@ -6,20 +6,21 @@
   "license": "MIT",
   "author": "",
   "scripts": {
-    "build": "vite build",
+    "build": "vite build && yarn prisma generate",
     "start": "nest start",
     "dev": "vite",
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
-    "console": "npm run start -- --watch --entryFile repl",
+    "console": "yarn start -- --watch --entryFile repl",
     "db:deploy": "npx prisma migrate deploy",
     "db:migrate": "npx prisma migrate dev",
     "db:seed": "npx prisma db seed",
+    "db:studio": "npx prisma studio",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "jest",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "test:e2e": "dotenv -e .env.test -- jest --config ./test/jest-e2e.json",
     "test:watch": "jest --watch"
   },
   "lint-staged": {
```

Comment:
This was news to me - when asking gpt it tells me this can be used similar to a pre-commit check, but in our case we mostly listed dependencies. How does this work?

Reply:
<img width="279" alt="image" src="https://github.com/mathiasberggren/streaming-subscription/assets/38131351/3d406ed0-0b7f-4135-849d-3dda5774b1a4">

Reply:
Nvm I just realised this did not come in this PR and I looked wrong on this during the morning when I was really tired 😅